content/en/hosting/requirements.md

```diff
@@ -1,49 +1,100 @@
 ---
-title: "Requirements"
+title: "CHT hosting requirements"
 linkTitle: "Requirements"
-weight: 5
+weight: 1
 aliases:  
   - /apps/guides/hosting/requirements
 description: >
-  Requirements for hosting CHT
+  Requirements for hosting CHT applications
 relatedContent: >
   hosting/3.x/self-hosting
   hosting/3.x/ec2-setup-guide
 ---
 
 {{% pageinfo %}}
-For production CHT deployments, Linux is recommended, with [Ubuntu](https://ubuntu.com/server) the most commonly used. For CHT development, Linux or macOS may be used. Windows can be used for either, but without recommendation.
+For production CHT deployments, Linux is recommended, with [Ubuntu](https://ubuntu.com/server) the most commonly used. For App Developer Hosting, Linux or macOS may be used. Windows can be used for either, but without recommendation.
 {{% /pageinfo %}}
 
-{{% alert title="Note" %}}
-These requirements apply to both 3.x and 4.x CHT hosting
-{{% /alert %}}
+Per the [Kubernetes vs Docker]({{< relref "hosting/kubernetes-vs-docker" >}}) page, CHT Core can be deployed with either Docker or Kubernetes.
+
+**Docker Compose**: 
+   * 4.x and 3.x Application developer
+   * 3.x Production.  Note: 3.x is [End-of-Life]({{< relref "core/releases#supported-versions" >}})
+
+**Kubernetes**: 
+   * 4.x Production
+## 3.x and 4.x Docker Compose App Developer Hosting
+
+
+* 4 GB RAM  / 2 CPU / 8 GB SSD
+* Root Access
+* TLS certificates -  Docker Helper for [3.x]({{< relref "hosting/3.x/app-developer#cht-docker-helper" >}}) or [4.x]({{< relref "hosting/4.x/app-developer#cht-docker-helper-for-4x" >}}) provides these  for you.
+* [Current version](https://docs.docker.com/engine/install/) of `docker` or current version of [Docker Desktop](https://www.docker.com/products/docker-desktop/) both of which include `docker compose`. Note that the older `docker-compose` has been [deprecated in favor of Compose V2](https://www.docker.com/blog/announcing-compose-v2-general-availability/). 
+
```

Comment:
Sorry! this was my mistake - there's a duplicate of this first title and set of bullet points - we should remove one:

```suggestion
```